common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -535,10 +535,20 @@ public static final class AAD {
     @NoArgsConstructor(access = AccessLevel.PRIVATE)
     public static final class SWITCH_BROWSER {
 
+        /**
+         * Path used on the broker redirect url to indicate the resume of the switch browser flow.
+         */
+        public static final String RESUME_PATH = "switch_browser_resume";
+
+        /**
+         * Path used on the broker redirect url to indicate the start of the switch browser flow.
+         */
+        public static final String START_PATH = "switch_browser";
+
         /**
          * String Query parameter key to indicate support for SWITCH_BROWSER protocol.
          */
-        public static final String PATH = "switch_browser";
+        public static final String CLAIM = "switch_browser";
```

Comment:
Why is this called CLAIM now?

Reply:
The previous constant called PATH was renamed to START_PATH (switch_browser) to differentiate it from RESUME_PATH (switch_browser_resume). The claim is the QP that indicates the client wants to initiate DUNA. Actually, I will rename it to QP.